libcontainer/factory.go

```diff
@@ -223,12 +204,10 @@ func (l *Factory) Load(id string) (*Container, error) {
 		newuidmapPath:        l.NewuidmapPath,
 		newgidmapPath:        l.NewgidmapPath,
 		cgroupManager:        cm,
+		intelRdtManager:      intelrdt.NewManager(&state.Config, id, state.IntelRdtPath),
 		root:                 containerRoot,
 		created:              state.Created,
 	}
-	if l.NewIntelRdtManager != nil {
```

Comment:
Perhaps squash this commit with the previous one (as it's refactoring the same parts of the code)

Reply:
Done (I was afraid it's too much to digest so split between two logical parts -- removing the interface and removing its usage).